index.html

```diff
@@ -39,7 +39,7 @@ <h2>Past Activities</h2>
 
   </main>
 
-</body>
+</body><script type="text/javascript" src="./Activity.js"></script>
```

Comment:
Not a major issue, but script tag needs to be moved down a row. Group agreed to address this during next iteration.